app/javascript/readingList/__tests__/readingList.test.jsx

```diff
@@ -18,7 +18,7 @@ describe('<ReadingList />', () => {
             body_text: 'Some body text',
             class_name: 'Article',
             path: '/bobbytables/what-s-in-your-database-2d3f',
-            published_date_string: 'Jun 22',
+            readable_publish_date_string: 'Jun 22',
```

Comment:
Search::FeedContent returns `readable_publish_date_string` rather than `published_date_string` so we need to use that in the view.